cmd/integration_link.go

```diff
@@ -464,3 +523,20 @@ func validateHoursBeforeDelete(ans interface{}) error {
 	}
 	return nil
 }
+
+func isAutomaticReviewAppFromForksStillAllowedAwaringUserOfSecurityConcerns() (bool, error) {
+	io.Warning(warningSecurityMessageForAutomaticReviewAppsFromForks)
+	forksAllowed := false
+	err := survey.AskOne(&survey.Confirm{
+		Message: "Automatic review apps creation from forks:",
+		Default: forksAllowed,
+	}, &forksAllowed, nil)
+	if err != nil {
+		return false, err
+	}
+	if forksAllowed {
+		io.Info("To bypass this security warning next time, you can provide the flag --aware-of-security-risks")
+	}
```

Comment:
I would not encourage in any way to bypass a security notice. User could find that option on their own easily as it is properly documented
```suggestion
```

Reply:
Well, I think if the user has already told he's been aware of the security risks, we could give him the way to bypass it, because he's already aware. IMHO it would be a better experience

Reply:
I would be in favor of security over UX in such case

Reply:
Also, the option to bypass the security warning is there mainly to prevent interactive mode in non-tty environment. Not to improve UX.

Reply:
Well if you insist, I'll go with it :)